content/articles/2019-06-rlang-0-4-0.Rmarkdown

```diff
@@ -0,0 +1,131 @@
+---
+title: 'rlang 0.4.0'
+author: Lionel Henry
+date: '2019-06-27'
+slug: rlang-0-4-0
+description: >
+  rlang 0.4.0 is now on CRAN!
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/ar7ZWQ-r87g
+  author: Caitlin Wynne
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>"
+)
+
+library(dplyr)
+```
+
+It is with great excitement that we announce the release of [rlang 0.4.0](https://rlang.r-lib.org) on CRAN. rlang is a toolkit for working with core R and Tidyverse features, and hosts the tidy evaluation framework. The full set of changes can be found in the [changelog](https://rlang.r-lib.org/news/index.html). In this article, we introduce the most important of these, the new tidy evaluation operator `{{`.
+
+
+## The good and bad of tidy evaluation
+
+Tidy eval powers packages like dplyr or tidyr. It makes it possible to manipulate data frame columns as if they were defined in the workspace:
+
+```{r, error = TRUE}
```

Comment:
I think the consensus was that we would call libraries visibly for each post (cc @jennybc). With dplyr, I usually give it its own chunk so I can `echo = FALSE` it.

Reply:
Done, thanks for reviewing!